00-HTML-CSS-basics/index_2.3.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta content="name=value,name=value" name="viewport" />
+    <meta charset="utf-8">
+    <link href="style_2.3.css" rel="stylesheet" type="text/css">
+
+ <!-- .......ITEM 6........ 
+    <style type="text/css">
+        
+        * {
+            font-size: 80px;
+        }
+
+    </style> -->
+
+    <title>CSS</title>
+</head>
+<body>
+
+<div class="wrapper">
+    <header class="header">
+        <font>header1</font>
```

Comment:
It's looks like you used `font` to override `font-size` value for item 6 and then you figured out that you can use asterisk for that...